### PR TITLE
Add a form validation to the portfolio

### PR DESCRIPTION
**Pull request by camilevahviraki**
### **In this milestone we did a form validation :**

- Edited the index.js file to add the validation .
- Update the index.html to call the validation function.
- Style the error message in css

### CODE REVIEW

**Review by nicupop729 — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️

Hi @camilevahviraki, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




### Some highlights for you
* Correct GitHub flow practice ✔️ 
* Good commit messages ✔️ 
* Clean codebase ✔️ 
* Linters are passing✔️

### Required Changes ♻️

At the moment, if I submit an empty form, still I receive feedback as "Well Done".

- [ ] Please correct this minor issue in order to pass this milestone and to have a nice functional form in your portfolio. :1st_place_medal: 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by camilevahviraki** [comment]
Hi @[nicupop729](https://github.com/nicupop729),
I understand what you say, but are you sure that this issue figures in the requirements sir?

File: index.html [hunk 1]
```diff
@@ -261,11 +266,11 @@ <h2 class="lng_elmnt">Terminal</h2>
                 name="message"
                 placeholder="Enter text here"
                 maxlength="500"
-                required
+                id="text_arr"
               ></textarea>
             </li>
             <li>
-              <button class="butn_submit" type="submit">Get in touch</button>
+              <button class="butn_submit" type="submit" onclick="return val();">Get in touch</button>
```
Comment (phelian23) [inline index.html:275]: - [ ] [Optional] A return statement should only be used within a function body (Javascript). Kindly remove it. I will suggest you make use of the event listener in your javascript file.

File: index.js [hunk 2]
```diff
@@ -465,3 +465,22 @@ function wrapProjectsCards() {
...
+  }
+  messageErr.textContent = 'Well done!';
+  messageErr.style.color = 'green';
+  setTimeout(() => {
+    messageErr.textContent = 'Well done!';
+  }, 3000);
+  return true;
+}
+
+val();
```
Comment (phelian23) [inline index.js]: - [ ] As requested by the previous reviewer, when an empty form is submitted, the message should be in the line of 'you cannot submit an empty form' and not 'well done'. Kindly fix it.

**Review by phelian23 — CHANGES_REQUESTED** [review]
Hi @camilevahviraki, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by JuliCarracedo — APPROVED** [review]
Hi @camilevahviraki , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Davidosky007 — COMMENTED** [review]
Hello @camilevahviraki  👋,

_This is an additional review as a result of the second opinion request._

- Kindly note that the requested change of correcting the feedback **"Well Done "** on empty input fields isn't part of the project requirements but a really nice to have feature, Though **OPTIONAL** i strongly suggest you either add a check to see if the input fields are not empty and all fields are filled correctly before the success message pops up or you take it out altogether as this would make your portfolio look professional and bug-free 👍  

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approved right away :)

Keep rocking 🚀!

**Pull request merged**